src/main/scala/org/apache/spark/streamdm/classifiers/trees/HoeffdingTree.scala

```diff
@@ -141,15 +146,16 @@ class HoeffdingTree extends Classifier {
       rdd =>
         val tmodel = rdd.aggregate(
           new HoeffdingTreeModel(model))(
-            (mod, example) => { mod.update(example) },
+          (mod, example) => { mod.update(example) },
           (mod1, mod2) => { mod1.merge(mod2, false) })
```

Comment:
mod1, mod2 is not a good variable name

Reply:
I've renamed the variable names in this portion of code. Thanks!